code-challenges.rb

```diff
@@ -31,3 +49,13 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome?(string)
+    if string.downcase == string.reverse.downcase
+       p "#{string} is a palindrome."
+    else
+       p "#{string} is not a palindrome."
+    end
+end
+
+palindrome? is_palindrome3
```

Comment:
I would be cautious about defining your method with a question mark. Another best practice is to make sure the `p` is on the method invocation.